Cab share/Login/LoginView.swift

```diff
@@ -101,7 +116,31 @@ struct LoginView: View {
             }
             .navigationBarHidden(true)
             .preferredColorScheme(.dark)
+//            .onAppear{
+//                Auth.auth().addStateDidChangeListener{ auth, user in
+//                    if user != nil {
+//                        userIsLoggedIn.toggle()
+//                    }}
+//            }
+        }
+    }
+    func signUp() {
+        Auth.auth().createUser(withEmail: email, password: password) {
+            result, error in
+            if error != nil {
+                print(error!.localizedDescription)
+            }
+        }
+        userSignedUp.toggle()
+    }
+    func signIn() {
+        Auth.auth().signIn(withEmail: email, password: password) {
+            result, error in
+            if error != nil {
+                print(error!.localizedDescription)
+            }
```

Comment:
Both func should be in a separate file. ViewModel file.

Reply:
you suggest to put in one file both functions ?

Reply:
maybe here there are some logical errors. Because in the tutorial he use .onappear to check the variable that allows us to change the view, instead I do it without checking. So we have to find a way to check an online state. I used another method because I wasn't able to manage the way that was shown in the video, but if you are able could be great